slim.Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM python:3.7-alpine as builder
+
+WORKDIR /girder
+RUN apk add --no-cache git gcc musl-dev libffi-dev openssl-dev
+COPY . ./
+# Both PYTHONDONTWRITEBYTECODE and --no-compile are necessary to avoid creating .pyc files
+RUN PYTHONDONTWRITEBYTECODE=1 pip install --no-cache-dir --no-compile .
+
+FROM python:3.7-alpine as runtime
+LABEL maintainer="Kitware, Inc. <kitware@kitware.com>"
+EXPOSE 8080
+
+# Set environment to support Unicode: http://click.pocoo.org/5/python3/#python-3-surrogate-handling
+ENV LC_ALL=C.UTF-8
+ENV LANG=C.UTF-8
+
+RUN apk add --no-cache tini
```

Comment:
Since `docker run` can be invoked with `--init`, do we really want to use `tini`?  It adds 100kb that education about docker options would make unnecessary.

Reply:
`--init` is not a well known feature of docker, and is not widely used.  100kb weighed against the 15 seconds of "oh right I have to --init this one container" that tini will regularly save me is, IMO, a no-brainer.

I'm all for education, but presently `--init` borders on the esoteric.

It's also worth noting that we don't always have the luxury of controlling args to docker run.  Suppose for example that docker-py doesn't support the `init` arg, or we try to deploy with Rancher and it doesn't expose that option, etc.   I'm not even certain the space savings is worth the amount of time it would take to look to see if it's supported.

Reply:
There's a customer satisfaction aspect here too.  For external customers,  even though they _can_ use `--init`, we probably shouldn't degrade their girder experience if they don't.

Reply:
See #3248